src/components/application_manager/src/commands/mobile/perform_audio_pass_thru_request.cc

```diff
@@ -120,6 +115,12 @@ void PerformAudioPassThruRequest::on_event(const event_engine::Event& event) {
 
   switch (event.id()) {
     case hmi_apis::FunctionID::UI_PerformAudioPassThru: {
+      LOG4CXX_INFO(logger_, "Received UI_PerformAudioPassThru");
```

Comment:
It lokks more like trace, because _receiving_ UI.  PAPT is not a part of PerformAudioPassThru _Request_ functionality.